drivers/memc/memc_mcux_flexspi.c

```diff
@@ -6,44 +6,44 @@
 
 #define DT_DRV_COMPAT	nxp_imx_flexspi
 
-#include <drivers/flash.h>
 #include <logging/log.h>
 #include <sys/util.h>
-#include "flash_mcux_flexspi.h"
+#include <drivers/memc/memc_mcux_flexspi.h>
 
-LOG_MODULE_REGISTER(flash_flexspi, CONFIG_FLASH_LOG_LEVEL);
+LOG_MODULE_REGISTER(memc_flexspi, CONFIG_MEMC_LOG_LEVEL);
 
-struct flash_flexspi_config {
+struct memc_flexspi_config {
 	FLEXSPI_Type *base;
 	uint8_t *ahb_base;
 	bool ahb_bufferable;
 	bool ahb_cacheable;
 	bool ahb_prefetch;
 	bool ahb_read_addr_opt;
 	bool combination_mode;
+	bool sck_differential_clock;
```

Comment:
nit: this should be added in commit `drivers: memc: Introduce i.MX RT FlexSPI HyperRAM driver`